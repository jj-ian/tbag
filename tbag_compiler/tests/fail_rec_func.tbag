/* Authors: Maria */
true {
  	print(fib(5));	
	endgame;
}

func int fib(int x) {
 	if (x < 2) { return 1;}
  	else { return fib("hello") + fib(x-2); }
}
