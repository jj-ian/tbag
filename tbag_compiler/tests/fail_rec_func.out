Fatal error: exception Failure("Function fib does
                                    not exist with the given parameters.")
